PredictiveMethods/CaseOutcome/PredictiveMonitor.py

```diff
@@ -65,4 +65,4 @@ def test(self, dt_test):
         for label_col_idx, label in enumerate(pred_model.cls.classes_):
             if label == self.pos_label:
                 # output probability of a positive class
-                print(np.round(predictions_proba[0, label_col_idx], decimals=3))
+                return np.round(predictions_proba[0, label_col_idx], decimals=3)
```

Comment:
Ideally, the transport between the Node.JS server and the PredictiveMethod python scripts would be abstracted out.  The main awkwardness in doing that is the changes to jobWorker.js.  In the master branch all the asynchronous details are tidily encapsulated within modelRunner's async.parallel block such that they actually look like a synchronous method call, whereas the kafka branch completely decouples incoming event handling in jobWorker-kafka.js from outgoing event+prediction handling in uiWorker.js.